pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -297,7 +297,6 @@ self: super: {
   hsbencher = dontCheck super.hsbencher;
   hsexif = dontCheck super.hsexif;
   hspec-server = dontCheck super.hspec-server;
-  HTF = dontCheck super.HTF;
```

Comment:
Wait, why do you remove this? This introduces a build error for this package. Just not changing this line could fix the HTF package. Or is there some reason I don‘t get? Maybe then you can even remove some of the `dontCheck`s below for which broken HTF is actually the reason? Or is HTF broken either way already before this PR? Anyway just let me now, what your rationale here was.

Reply:
I think this was a mistake, I see no rationale for removing it. Sending in a new version of this PR.